### ISSUE TITLE
Help: Cannot get dynamic blocks to work

### ISSUE THREAD

**Issue by lyndhurst** [ahmed-shariff/org-roam-ql#11]
Hi, 

first of all thank you for this package, I already have plenty of uses planned for it.

I am new to Emacs (Doom Emacs actually) so there is something dumb I must be missing. I installed the package without any problem, and `org-roam-ql-search` works really well.

I tried to build a simple dynamic block in an `org-roam` file in the `org-roam-directory`, but nothing happens. I looked around, and I am not finding a command to refresh or evaluate the block (I even tried some babel and `eval` commands in `M-x` to no avail).

Here is my block:

```org
#+BEGIN: org-roam-ql :query (tags "draft") :columns (title tags)

#+END:
```

I also tried without a colon at the `#+END` since between the project page and the issues I have been reading, I saw both syntaxes being used. I even tried as a source block.

Since I was able to make `custom-agenda-views` work with `org-roam-ql-agenda-block`, and some custom wrappers to invoke  `org-roam-ql-search` buffers directly from custom key bindings, I do not know how to look further.

Thanks in advance for your time.

**Comment by ahmed-shariff** [comment]
Thank you very much for the feedback. It's nice to know others are getting use out of this :)

The block you have should work. It can be in any org file, it doesn't have to be in the org-roam-directory for the dynamic block to work - I even use it in scratch buffers.

To clarify, you create the block, move into it and do `C-c C-c` and there is no result?
Also, are you on the latest commit of org-roam-ql?
Also, when you run `org-roam-ql-search` with the query `(tags "draft")` do you get any results?

**Comment by lyndhurst** [comment]
I knew it was something obvious for most users, but I did not use `C-c C-c` ! I actually never used it before, and just found out thanks to your comment how useful it is. I made it so far without even using the 'do-it-all' binding still being amazed at how ergonomic orgmode is :)

You are welcome for the feedback, I gotta say I managed to put together stuffs I was really frustrated about with my previous zettelkasten setup. So far I put together some maintenance queries I have never been able to achieve in the last four years or so, and I did it using your package in a day.

For a newbie like me that has maybe a couple of month of experience with Emacs (mostly configuring stuffs) I managed this morning to add three predicates to help with maintenance: filter nodes comparing tags length, atime and mtime (not sorting). It was a breeze by copying and rearranging some of your code from the package source file, and it is incredibly valuable to maintain well groomed notes.

So thanks again for the reply, and the package because I always felt notes metadata was pretty useless without a handy way to retrieve your notes with it, and now this is fixed.
